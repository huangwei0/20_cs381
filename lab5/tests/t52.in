findall(Z, aunt('Queen Elizabeth II', Z), Y), 
print(Y), 
print('\n'), 
halt.
