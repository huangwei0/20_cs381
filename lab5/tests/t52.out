[David: Viscount Linley,Sarah Armstrong-Jones]
